src/notebooks/controllers/ipywidgets/message/commonMessageCoordinator.ts

```diff
@@ -83,63 +83,49 @@ export class CommonMessageCoordinator {
         this.ipyWidgetScriptSource?.dispose(); // NOSONAR
     }
 
-    public async attach(webview: IWebviewCommunication) {
```

Comment:
This was async, but if you check the call path to this method, we don't await at all, hence unnecessarily async.
Also, the webview is assumed to be always one-one for notebooks, but we can have multiple and that's why we have an attach method else it could have been passed via CTOR..

Looks like the code was broken after some refactoring.